app/javascript/dashboard/modules/contact/components/ContactAttribute.vue

```diff
@@ -8,7 +8,26 @@
         {{ label }}
       </h4>
     </div>
-    <div class="value-wrap">
+    <div v-show="isEditing">
```

Comment:
A comment on the implementation. Should we use content-editable instead of input box? Something like this https://www.npmjs.com/package/vue-contenteditable

Reply:
This looks like good improvement over the current one. I'll pickup this an improvement in a new PR.